### DISCUSSION TITLE
Consider upgrading protobuf API

### DISCUSSION THREAD

**Discussion by leungyauming** [v2fly/v2ray-core#669] [Ideas]
From https://blog.golang.org/protobuf-apiv2, the second version of protobuf API (google.golang.org/protobuf) is the replacement of the first version of protobuf API (github.com/golang/protobuf).

Will the team consider moving to the new version of protobuf API?
If yes, I may try to work on it during spare time.

**Comment by Loyalsoldier** [comment]
See the branch `v5`.

**Reply by leungyauming** [reply]
I can see that the protobuf API is already upgrade in branch `v5`.
Sorry for the duplicated suggestion.
